functions/venue.js

```diff
@@ -842,7 +842,19 @@ exports.updateVenueNG = functions.https.onCall(async (data, context) => {
   // @debt updateVenue uses checkUserIsOwner rather than checkUserIsAdminOrOwner. Should these be the same? Which is correct?
   await checkUserIsOwner(data.id, context.auth.token.user_id);
 
-  const updated = {};
+  if (!data.worldId) {
+    throw new HttpsError(
+      "not-found",
+      "World id is missing and the update can not be executed."
+    );
+  }
+
+  const updated = {
+    config: {
+      landingPageConfig: {},
+    },
+  };
```

Comment:
Isn't there a Lodash merge object utility we can use in this and similar cases?

Reply:
There should be, yes. But ideally this should look like the world object with either `isNil` or `isEmpty`

Reply:
More like https://lodash.com/docs/#merge that does what the Firestore merge flag does, only inside the function, before calling the cloud API:

> like _.assign except that it recursively merges own and inherited enumerable string keyed properties of source objects into the destination object